plugin/src/main/java/io/jenkins/plugins/casc/SecretSourceResolver.java

```diff
@@ -25,9 +26,14 @@ public static String resolve(ConfigurationContext context, String toInterpolate)
     }
 
     private static String handle(ConfigurationContext context, String captured) {
-        return reveal(context, captured)
+        String[] split = captured.split(defaultDelimiter, 2);
```

Comment:
This seems a bit cleaner, and easier to read:
```
        String[] split = captured.split(defaultDelimiter, 2);
        return Tuple.of(split[0], Try.of(() -> split[1]).toJavaOptional())
                .apply((toReveal, defaultValue) -> reveal(context, toReveal)
                        .map(Optional::of)
                        .orElse(defaultValue)
                        .orElse(""));
```

Reply:
Great suggestion :smiley: